lib/exercises.rb

```diff
@@ -1,19 +1,48 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n*m), when build the hash table, it's O(n) (n is the size of the list), 
+#                          and the inner sort depends on the lenght of the word, 
+#                          so it will be O(n)*O(m) --> O(n*m).
+# Space Complexity: O(n), the worst case scenarios for hash table and result are O(n).
+#                         So it will be O(n)*2 --> O(n) (n is the size of the list).
 
-def grouped_anagrams(strings)
-  raise NotImplementedError, "Method hasn't been implemented yet!"
+def grouped_anagrams(list)
```

Comment:
👍 The sort is `m log m` so your complexities are off, but if you assume the words are limited in length the time complexity is O(n)